lib/filesharing/files_usecases/lib/src/file_viewer/widgets/file_page_app_bar.dart

```diff
@@ -54,7 +54,10 @@ class _Title extends StatelessWidget {
   Widget build(BuildContext context) {
     return Text(
       name,
-      style: const TextStyle(color: Colors.white, fontFamily: rubik),
```

Comment:
Just a note for future PRs: Would be great to only have changes in a PR that fit the purpose of the PR. This purpose of the PR is to refactor the imports of the `sharezone_widgets` package and should not contain any formatting changes like this.

Reply:
I mean I guess thats the same kind of auto-formatting that would happen if we save the file again to edit the imports ourselves, no? So I don't know if thats the case I don't have a problem with it really 🤷‍♂️   
Or is it different formatting from ours?